src/color.scss

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2020 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+/* stylelint-disable scss/dollar-variable-empty-line-before, at-rule-empty-line-before, order/properties-order, scss/selector-nest-combinators, declaration-empty-line-before, 
+   scss/at-extend-no-missing-placeholder,  declaration-no-important, selector-max-combinators, selector-max-compound-selectors, no-descending-specificity, color-function-notation */
+
+::slotted(*) {
+  color: var(--ds-auro-popover-text-color);
+}
+
+.popover {
+  background-color: var(--ds-auro-popover-background-color);
```

Comment:
this should be a 'container' token not a 'background' token. I know we get to make up our own tier 3 tokens here but we should follow the semantic name of the tier 2 in this sense.